### PR TITLE
feat: implement contributor list view

### LINKED ISSUE #1155: feat: Implement List View on contributors tab
This requires the following before be started

https://github.com/open-sauced/insights/issues/1153
https://github.com/open-sauced/insights/issues/1154

Screen Shot 2023-05-08 at 10 05 28 PM

## Design 

[Figma](https://www.figma.com/file/768ifqOGWrsRN29Igkc4ip/User-Flows?type=design&node-id=205-1661&t=eleETzJqGQfeLI12-0)

### PR DESCRIPTION
**Pull request by OgDev-01**
## What type of PR is this? (check all applicable)

- [ ] 🍕 Feature
- [ ] 🐛 Bug Fix
- [ ] 📝 Documentation Update
- [ ] 🎨 Style
- [ ] 🧑‍💻 Code Refactor
- [ ] 🔥 Performance Improvements
- [ ] ✅ Test
- [ ] 🤖 Build
- [ ] 🔁 CI
- [ ] 📦 Chore (Release)
- [ ] ⏩ Revert

## Description
This PR includes two changes. Firstly, it renames the file extension for better organization. Secondly, it adds a list view contributor table for better user experience. These changes improve the codebase and enhance the functionality of the application.

_Generated using [OpenSauced](https://opensauced.ai/)._

Mobile view is not looking all nice, it can be improved in the future :<

## Related Tickets & Documents
Closes #1155 

### Grid
image

### List 
image
image

## Added tests?

- [ ] 👍 yes
- [x] 🙅 no, because they aren't needed
- [ ] 🙋 no, because I need help

## Added to documentation?

- [ ] 📜 README.md
- [ ] 📓 docs.opensauced.pizza
- [ ] 🍕 dev.to/opensauced
- [ ] 📕 storybook
- [x] 🙅 no documentation needed

### CODE REVIEW

File: components/molecules/ContributorListTableRow/contributor-list-table-row.tsx [hunk 1]
```diff
@@ -0,0 +1,164 @@
...
+
+function getLastContributionDate(contributions: DbRepoPR[]) {
+  if (contributions.length === 0) {
+    return "-";
+  }
+  const sortedContributions = contributions.sort((a, b) => {
+    return +new Date(b.merged_at) - +new Date(a.merged_at);
+  });
+
+  return getFormattedDate(sortedContributions[0]?.merged_at);
```
Comment (bdougie) [inline components/molecules/ContributorListTableRow/contributor-list-table-row.tsx:29]: This should display as "2 days ago" and not the full date.

Comment (OgDev-01) [reply components/molecules/ContributorListTableRow/contributor-list-table-row.tsx:30]: Ok...

**Comment by OgDev-01** [comment]
@bdougie  
image

image

**Comment by bdougie** [comment]
image

Perhaps it's because company section is blank.

Can do sometime to display text of the last repo they contributed to?

I.e. open-sauced/insights?

**Comment by OgDev-01** [comment]
Update @bdougie, the requested changes have been added 👍

**Comment by bdougie** [comment]
> Update @bdougie, the requested changes have been added 👍

Mobile view will need grey text as well
Screen Shot 2023-05-24 at 5 16 58 PM

**Review by bdougie — APPROVED** [review]

**Pull request merged**